src/core/contract_address.ml

```diff
@@ -0,0 +1,11 @@
+open Tezos
+open Crypto
+
+type t = Contract_hash.t [@@deriving yojson, eq, ord]
+
+(* contract address is a hash of user_operation hash *)
+let of_user_operation_hash (hash : BLAKE2B.t) =
+  hash |> BLAKE2B.to_raw_string |> BLAKE2B_20.hash
+
+let to_string = Contract_hash.to_string
```

Comment:
This is not okay, as we're adding a proper abstraction for Contract_address, this should have it's own prefix.